### ISSUE TITLE
SyntaxError

### ISSUE THREAD

**Issue by musicformellons** [makinacorpus/django-mbtiles#27]
I get:
python3.4/site-packages/mbtilesmap/models.py", line 54
    except (AssertionError, InvalidFormatError), e:
                                               ^
SyntaxError: invalid syntax

The ^ is pointed at the e.

**Comment by leplatrem** [comment]
True, it should be written ’as e’ in python 3. Feel free to open a pull-request !
